nano/secure/pending_info.hpp

```diff
@@ -27,10 +28,20 @@ class pending_info final
 	size_t db_size () const;
 	bool deserialize (nano::stream &);
 	bool operator== (nano::pending_info const &) const;
-	nano::account source{};
-	nano::amount amount{ 0 };
-	nano::epoch epoch{ nano::epoch::epoch_0 };
+	nano::account source{}; // the account sending the funds
+	nano::amount amount{ 0 }; // amount receivable in this transaction
+	nano::epoch epoch{ nano::epoch::epoch_0 }; // epoch of sending block, this info is stored here to make it possible to prune the send block
+
+	friend std::ostream & operator<< (std::ostream & os, const nano::pending_info & info)
```

Comment:
This obviously won't work well with tracing or serialization, not sure how useful it is aside from primitive test logging.

Reply:
It is mainly for use for primitive logging and for printing the value of the struct from inside the debugger.
I am happy to make it compatible with tracing library but I would like to do that as a separate step so that it doesn't become a big task.